foncier/templates/sorry.html

```diff
@@ -2,12 +2,13 @@
 <html lang="fr">
     <head>
         <meta charset="UTF-8">
+        <link rel="stylesheet" href="static/styles.css">
```

Comment:
I only use `../static` on retrieve page : `/retrieve/6465-684-654-6-468-48` because we need to search `static` folder in parent folder of `retrieve`. So it should work with current code.